app/full_text_search.py

```diff
@@ -125,6 +150,11 @@ def search(repo, query_txt, max_results):
     # Remove double quotes so that we can safely apply enclose_in_double_quotes().
     query_txt = re.sub('"', '', query_txt)
     romanized_query = create_query_txt(query_txt)
+    kanji_words_in_query_txt = script_variant.find_kanji_word(query_txt)
+    double_quote_kanji_words_in_query_txt = ' '.join(
+        enclose_in_double_quotes(word) for word in kanji_words_in_query_txt)
+    romanized_query_with_kanji = romanized_query + ' ' +\
```

Comment:
Do you need to add romanized_query? Would it work if you search just with double_quote_kanji_words_in_query_txt?